tests/unit/test_infos.py

```diff
@@ -201,7 +203,7 @@ def test_project_info_set_project_raw_write():
 def test_project_info_set_project_var_bad_name():
     info = ProjectInfo(application_name="test", cache_dir=Path())
 
-    with pytest.raises(ValueError) as raised:
+    with pytest.raises(ValueError) as raised:  # noqa: PT011
```

Comment:
I'd like @cmatsuoka's opinion here. This is complaining that we don't have a `match=` in the context manager. We're comparing the string below instead. I think the options are:

1. Keep it as is, so the lint error reminds us to check equality
2. Move to using `match`
3. Silence PT011 for `craft-parts` (and possibly in all our repos)